std/experimental/ndslice/iteration.d

```diff
@@ -104,6 +104,8 @@ import std.meta;
 import std.experimental.ndslice.internal;
 import std.experimental.ndslice.slice; //: Slice;
 
+@trusted:
```

Comment:
`iteration` module must manage only save operations on strides and lengths. It must not contain any other stuff.